src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write code here
+const header = document.querySelectorAll('thead tr th');
+
+[...header].map((element, index) => {
```

Comment:
Why you use map method? You don't need to return a new array, please use foreach